docs/open-payments.md

```diff
@@ -0,0 +1,7 @@
+# Open Payments
+
+Open Payments is an API standard that allows third-parties (with the account holder's consent) to initiate payments and to view the transaction history on the account holder's account. Extensive documentation can be found on the [Open Payments website](https://openpayments.guide).
+
+Rafiki implements the Open Payments APIs. Every request and response is validated against the [Open Payments API specification](https://github.com/interledger/open-payments/tree/main/openapi).
+
+To update the API specification, create a Pull Request on the [Open Payments Github Repository](https://github.com/interledger/open-payments/). Changes will automatically be pushed to [Readme](https://readme.com/), which hosts the [API reference](https://docs.openpayments.guide/reference/) on the [Open Payments website](https://openpayments.guide).
```

Comment:
Maybe reference where Rafiki Open Payments "version" is configured
https://github.com/interledger/rafiki/blob/d33d4d9b5c3423e7990d0151647062e78942d53b/packages/open-payments/scripts/fetch-schemas.sh#L12-L14